alabaster/static/alabaster.css_t

```diff
@@ -221,6 +221,14 @@ div.sphinxsidebar hr {
     width: 50%;
 }
 
+div.sphinxsidebar .badge {
+    border-bottom: none;
+}
+
+div.sphinxsidebar .badge:hover {
+    border-bottom: none;
+}
+
```

Comment:
I think this whole commit is just working around the root cause, which I fixed in #125 almost a year ago but which still needs to be merged.

(I think the root cause is that some CSS  in the theme that intends to clear the bottom border for all images is accidentally getting overridden by some CSS that comes after it.)

#125 has gotten 👍, but still has not been merged. Once it is merged, I think this workaround can be removed.

/cc @bitprophet @sloria

Reply:
Hi @bitprophet, my previous comment is still the case, so I just thought I'd see if you might have a moment to merge my PR. Thanks for taking a look if you can!

Reply:
(#125)